src/test/java/org/broadinstitute/consent/http/resources/ElectionResourceTest.java

```diff
@@ -125,6 +128,20 @@ public void testIsDataSetElectionOpenError() {
         Assert.assertEquals(ERROR, response.getStatus());
     }
 
+    @Test
+    public void testDescribeVotesOnElection() {
+        Response response = electionResource.describeVotesOnElection(authUser, randomInt());
+        Assert.assertEquals(OK, response.getStatus());
+    }
+
+    @Test
+    public void testDescribeVotesOnElectionError() {
+        when(voteService.findVotesByElectionId(any())).thenThrow(new NotFoundException());
+        electionResource = new ElectionResource(voteService, electionService);
+        Response response = electionResource.describeVotesOnElection(authUser, any());
+        Assert.assertEquals(NOT_FOUND, response.getStatus());
+    }
```

Comment:
The reason this test works is because we're telling `voteService.findVotesByElectionId` to throw an exception that it really doesn't throw in the current implementation.